### PR TITLE
fix : docs layout and docs css

### PR DESCRIPTION
**Pull request by sushil-mod**
Fixes: modified documentation layout

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (psychoactiv) [inline .vscode/settings.json]: Remove this vscode folder

File: Documentation/document.css [hunk 2]
```diff
@@ -1,76 +1,130 @@
...
-    flex-direction: row;
-    gap: 2rem;                                                      
-}
-.doc-badge-avatar{
-    display: flex;
-    flex-direction: row;
-    gap: 2rem; 
+.component-show{
+    gap: 2rem;  
+    background-color: white;
```
Comment (psychoactiv) [inline Documentation/document.css]: you could use color variables

**Review by psychoactiv — APPROVED** [review]
Do these minor changes, code is good to go

File: .vscode/settings.json (hunk 1 above)
Comment (sushil-mod) [reply .vscode/settings.json]: done

File: Documentation/document.css (hunk 2 above)
Comment (sushil-mod) [reply Documentation/document.css]: done , thank you ,  i missed

File: index.html [hunk 3]
```diff
@@ -1,39 +1,606 @@
...
+        <li class="aside-li"><a href="#image" class="aside-a">Image</a></li>
+        <li class="aside-li"><a href="#text" class="aside-a">Text</a></li>
+
+
+        <li class="aside-li"><a href="#" class="aside-a">sample1</a></li>
+        <li class="aside-li"><a href="#" class="aside-a">sample2</a></li>
+        <li class="aside-li"><a href="#" class="aside-a">sample3</a></li>
+        <li class="aside-li"><a href="#" class="aside-a">sample4</a></li>
+    
+
```
Comment (Kothalkarpriya) [inline index.html:51]: remove extra white spaces from code

File: index.html [hunk 4]
```diff
@@ -1,39 +1,606 @@
...
+
+
+      </div>
+      </div>
+
+      <!-- alert  -->
+      <div class="component-div">
+
+  <p id="alert" class="doc-header">Alert</p>
```
Comment (Kothalkarpriya) [inline index.html:180]: Do use Code formatting

File: index.html [hunk 5]
```diff
@@ -1,39 +1,606 @@
...
-    <main class="main-container">
+        <iframe src="https://www.thiscodeworks.com/embed/6207ca133d90130015398c28" style="width: 70%; height: 200px;"
+          frameborder="0"></iframe>
+
+        <p class="doc-label">Card with text over image</p>
+        <!-- text-over-card  -->
+
+        <div class="component-show">
+<div class="card card-pos-rel">
+          <img class="card-img " src="https://images.unsplash.com/photo-1536323760109-ca8c07450053"
```
Comment (Kothalkarpriya) [inline index.html:343]: include / before closing image tag

**Review by Kothalkarpriya — APPROVED** [review]
Look into my comments, after fixing them you can merge

File: index.html (hunk 3 above)
Comment (sushil-mod) [reply index.html:51]: done

File: index.html (hunk 4 above)
Comment (sushil-mod) [reply index.html:180]: done

File: index.html (hunk 5 above)
Comment (sushil-mod) [reply index.html:343]: done thank you

**Pull request merged**